### ISSUE TITLE
Ambassador Task: Write an opinion article or blog about your experience creating a token on TokenMint on testnet

### ISSUE THREAD

**Issue by PowerVANO** [HorizenOfficial/HDE-Proposals#65]
**Scope of work:**

This article can be any length but must cover:
- What is TokenMint	
- How to get test TZEN
- How to get Testnet Sphere wallet and Testnet Cobalt wallet
- Where to find documentation and resources
- The steps required to create a token

**Schedule Milestones:**

This task may only be completed by an official Horizen Ambassador or pre-approved person. For the task to be considered complete, it will need to be reviewed and approved by the Horizen team. The team may request edits be made to the article before approval.

If you are not an ambassador and would like to be approved for this task, contact erica [at] horizen.io

Ambassadors that accept this task must provide bi-weekly updates during the period they are working on the task.

**Budget: $50 in ZEN per person / Total of $300 (up to 6 participants)**

**Comment by mahnoor089** [comment]
Hello :)
I'm interested in unleashing more about TokenMint and guide new users.
I would like to prepare an article which briefly explain all the factors mentioned in the issue field.
Kindly let me know if I'm eligible to work on this.

**Comment by paragismb** [comment]
Can I take up this job?

**Comment by PowerVANO** [comment]
@paragismb yes, you can. I do not believe we have 6 people already working on this.

**Comment by paragismb** [comment]
> @paragismb yes, you can. I do not believe we have 6 people already working on this.

Tokenmint chain is under maintenance. I can do it once it is active I think.

**Comment by paragismb** [comment]
Work done. Article Link: https://thecoinbuzz.substack.com/p/create-your-own-token-using-horizen
Tweet: https://twitter.com/paragism_/status/1543253676248035328?s=20&t=jAitrFHyhxi1zhldbdOKGQ

**Comment by PowerVANO** [comment]
Payment claim submitted and is being processed.

**Comment by PowerVANO** [comment]
@paragismb @Horusfeathers payment has been issued.
